### ISSUE TITLE
Cannot connect to Bosh Director

### ISSUE THREAD

**Issue by bespokeeagle** [cloudfoundry/bosh#2503]
Hello i have a bosh director running on a fips stemcell and i am currently unable to communicate with it

I have a feeling it might be a tls issue, when i check the logs i see the following

`SSL_do_handshake() failed (SSL: error:1C800073:Provider routines::invalid data error:0A0C0103:SSL routines::internal error`

i generated the certificates using the bosh interpolate command and also tried generating with openssl both produce this same error

any ideas???

**Comment by beyhan** [comment]
What is the version of the BOSH Director and stemcell do you use? The BOSH Director version https://github.com/cloudfoundry/bosh/releases/tag/v280.0.15 is required an also, the BOSH CLI version you require is at least `7.4.1`.

**Comment by bespokeeagle** [comment]
> What is the version of the BOSH Director and stemcell do you use? The BOSH Director version https://github.com/cloudfoundry/bosh/releases/tag/v280.0.15 is required an also, the BOSH CLI version you require is at least `7.4.1`.

Ah okay, let me update these then

**Comment by bespokeeagle** [comment]
@beyhan i have updated the bosh director and i have switched back to the latest jammy stemcell and now i am not able to create a release, it fails because it cannot reach compilation vms

**Comment by jpalermo** [comment]
Probably a network or certificate problem. You'll need to pull the agent logs from the compilation VM to see what's wrong.

https://bosh.io/docs/tips/#unresponsive-vm-logs

Compilation VMs are often tricky because they get destroyed. You might instead try creating a deployment with just a single VM and no jobs so it's easier to grab the agent logs.

**Comment by bespokeeagle** [comment]
@jpalermo yes getting the logs from the compilation vms was a bit tricky but i was able to, turns it was a nats authentication problem, apparently there are some breaking changes between my old bosh director release and the latest one

> Probably a network or certificate problem. You'll need to pull the agent logs from the compilation VM to see what's wrong.
> 
> https://bosh.io/docs/tips/#unresponsive-vm-logs
> 
> Compilation VMs are often tricky because they get destroyed. You might instead try creating a deployment with just a single VM and no jobs so it's easier to grab the agent logs.

@jpalermo yes getting the logs from the compilation vms was a bit tricky but i was able to, turns it was a nats authentication problem, apparently there are some breaking changes between my old bosh director release and the latest one
